src/mmw/js/src/draw/controllers.js

```diff
@@ -27,6 +29,8 @@ var DrawController = {
         App.rootView.geocodeSearchRegion.show(geocodeSearch);
         App.rootView.drawToolsRegion.show(toolbarView);
 
+        enableSingleProjectMode();
```

Comment:
This function's name seems to imply that Single Project Mode is _always_ enabled. Consider renaming to `enableSingleProjectModeIfItsi`, or to place the condition here instead of within the function:

```javascript
if (coreUtils.getParameterByName('initialize_itsi') === 'true') {
    enableSingleProjectMode();
}
```